packages/wonder-blocks-tokens/src/theme/primitive/border.ts

```diff
@@ -12,6 +12,7 @@ export const border = {
      */
     radius: {
         radius_0: remToPx(sizing.size_0),
+        radius_010: remToPx(sizing.size_010),
```

Comment:
Do we want an equivalent in Figma for radius_010? 

<img width="326" height="415" alt="Image" src="https://github.com/user-attachments/assets/3f748e49-dfc0-405f-9815-4b1fc6f7e3bc" />

Reply:
I'm seeing that token already in the Foundations Figma file:
<img width="1384" height="1237" alt="Screenshot 2025-08-12 at 1 54 52 PM" src="https://github.com/user-attachments/assets/01da3b9c-d4cc-427f-8173-3c2e9d9d1599" />

Reply:
I see the Figma file has a token from `sizing`, I posed it to Caitlyn since I'm a read-only consumer of the Figma files!